ModelicaTest/MultiBody.mo

```diff
@@ -3734,14 +3740,14 @@ a linear damper is connected here.
         v(fixed=true),
         n={0,1,0},
         s(start=-1, fixed=true),
-        a(fixed=true, start=-0.81)) annotation (Placement(transformation(
+        a(fixed=true, start=9-world.g)) annotation (Placement(transformation(
```

Comment:
@MartinOtter As I fixed the assert-clause below, it was triggered. So I changed `prismatic1.a.start` to far more precise value to fulfill the assertion. If this was not originally intended in the test model, then some other estimation shall be taken. But the original `start=-0.81` is too hight.

Reply:
I was a bit lost here. Having -0.81 as a magic number is weird, changing it to 9-world.g just seems like magic!
After some thinking and translating the model in Dymola I got that the model has the following parameters:

  ```
  parameter Modelica.Units.SI.TranslationalSpringConstant spring_c=10
    "Spring constant";
  parameter Modelica.Units.SI.Length spring_s_unstretched=0.1
    "Unstretched spring length";
  parameter Modelica.Units.SI.Position prismatic_s_start=-1
    "Relative distance between frame_a and frame_b";
  parameter Modelica.Units.SI.Mass m=1;
```

Adding them where appropriate allows us to write:
`a(fixed=true, start=spring_c*(-prismatic_s_start-spring_s_unstretched)/m - world.g)`

I don't know if there is an intuitive explanation for the formula, but at least it seems to reduce the magic (and not only does it work, it's also unit-correct).

Reply:
@HansOlsson Perfect, I like it! (Why didn't I just think of that.)

The question remains - was it an intention to cause the difference for the assert? Or is that precise value just fine now?